### PR TITLE
Csp 973 create event check your answers

### PR DESCRIPTION
**Pull request by MarkFCain**

### CODE REVIEW

File: src/SFA.DAS.Admin.Aan.Web/Views/ManageEvent/CheckYourAnswers.cshtml [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+                        <h2 class="govuk-heading-m">
+                            Guest speakers
+                        </h2>
+                    </dt>
+                    <dd class="govuk-summary-list__value">
+                    <ol class="govuk-list govuk-list--number">
+                        @foreach (var speaker in Model.GuestSpeakers)
+                        {
+                            <li>
+                                @speaker.GuestName.Trim(), @speaker.GuestJobTitle.Trim()
```
Comment (DevenKShah) [inline src/SFA.DAS.Admin.Aan.Web/Views/ManageEvent/CheckYourAnswers.cshtml:203]: Are we not trimming where we are taking the input? If we ensure this then there is no need to use Trim anywhere else.

File: src/SFA.DAS.Admin.Aan.Web/Controllers/ManageEvent/CheckYourAnswersController.cs [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+    [HttpPost]
+    public IActionResult Post()
+    {
+        return RedirectToRoute(RouteNames.ManageEvent.CheckYourAnswers);
+    }
+
+
+    private async Task<CheckYourAnswersViewModel> GetViewModel(EventSessionModel sessionModel, CancellationToken cancellationToken)
+    {
+
```
Comment (SarahDuncan) [inline src/SFA.DAS.Admin.Aan.Web/Controllers/ManageEvent/CheckYourAnswersController.cs:44]: can we delete this whitespace?

**Review by SarahDuncan — APPROVED** [review]

File: src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+    public bool? IsAtSchool { get; set; }
+
+    public string? OrganiserName { get; set; }
+    public string? OrganiserEmail { get; set; }
+
+    public int? NumberOfAttendees { get; set; }
+
+    public bool ShowLocation =>
+        EventFormat is Application.Constants.EventFormat.InPerson or Application.Constants.EventFormat.Hybrid;
```
Comment (DevenKShah) [inline src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs:37]: Application.Constants is already in the namespace, is there a collision?

File: src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+
+    public bool ShowOnlineEventLink =>
```
Comment (DevenKShah) [inline src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs:39]: This is missing coverage?

File: src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
+        EventFormat is Application.Constants.EventFormat.Online or Application.Constants.EventFormat.Hybrid;
+
+    public string GetDateAndTimeFormatted()
+    {
+        if (Start == null || End == null) return "";
+
+        var startDateTimeFormatted = Start.Value.Minute == 0
+            ? Start.Value.ToString("htt").ToLower()
```
Comment (DevenKShah) [inline src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs:47]: Was it in requirements to not show succeeding `00` for mins?

**Review by DevenKShah — APPROVED** [review]
Few questions, but otherwise looks good

File: src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs (hunk 5 above)
Comment (MarkFCain) [reply src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs:47]: No but I took my lead from screenshots. I'll remove the complexity though, it's not specified

File: src/SFA.DAS.Admin.Aan.Web/Views/ManageEvent/CheckYourAnswers.cshtml (hunk 1 above)
Comment (MarkFCain) [reply src/SFA.DAS.Admin.Aan.Web/Views/ManageEvent/CheckYourAnswers.cshtml:203]: I'll change it to the input

File: src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs (hunk 4 above)
Comment (MarkFCain) [reply src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs:39]: No there is coverage, I just combined tests.  I will separate to make it clearer

File: src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs (hunk 3 above)
Comment (MarkFCain) [reply src/SFA.DAS.Admin.Aan.Web/Models/ManageEvent/CheckYourAnswersViewModel.cs:37]: yes, there is a field in the viewModel called EventFormat

File: src/SFA.DAS.Admin.Aan.Web/Views/ManageEvent/CheckYourAnswers.cshtml (hunk 1 above)
Comment (DevenKShah) [reply src/SFA.DAS.Admin.Aan.Web/Views/ManageEvent/CheckYourAnswers.cshtml]: @MarkFCain we need to Trim all the text inputs that we are capturing in creation journey.

**Review by DevenKShah — APPROVED** [review]

**Pull request merged**